onboarding.md

```diff
@@ -0,0 +1,10 @@
+# Onboarding
+
+DLS strives to be welcoming and inclusive in our onboarding process for new team members. The initial tasks for any new team member will include:
+
+- Becoming familiar with our team and department handbooks and Library goals
+- Setting up hardware, including a development environment and password management
+- Creating accounts across a variety of services
+- Work through training curriculum, if relevant, and pair with DLS colleagues
```

Comment:
We talked about how to connect folks who are in training with real-world tickets (solo or pair) that may be related to their training. How do we facilitate this? Do we want to mention anything about that intention in here?

Reply:
Hm, yeah, I kept this pretty vague because this page covers an area that is in flux for a number of reasons (we don't do it very often, it has to be pretty flexible / individualized).

For example, if we hire a mid or senior level developer, they're less likely to have a training curriculum per se and more likely to go straight to "real world" tickets.

But I do think that's what I was trying to imply when I said "pair with DLS colleagues." We could say something like "pair with DLS colleagues on projects or github tickets"

Feel free to suggest language!

Reply:
@hackartisan I think this wording would clarify it:

`Working through training curriculum, if relevant; Reaching out to DLS colleagues as lessons are completed for help finding tickets and pairing opportunities that may help reinforce the lessons.`

Reply:
This language will be more relevant in a training curriculum/plan. We can move forward without it.